### PR TITLE
Add brown bread and bread flour with container

### PR DESCRIPTION
**Pull request by Xaleth**
#### Summary
 SUMMARY: Content "Add brown bread and bread flour"

#### Purpose of change
Moar content

#### Testing
They spawn

#### Additional context
A step into adding canned bread, a staple New England food.

### CODE REVIEW

File: .gitmessage [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# If this commit is applied, it will ....
```
Comment (Xaleth) [inline .gitmessage:1]: oops, not sure how I got this deleted

File: data/json/recipes/recipe_food.json [hunk 2]
```diff
@@ -4468,6 +4468,30 @@
...
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": "5 m",
+    "charges": 20,
+    "autolearn": true,
+    "batch_time_factors": [
+      [ [ "flour", 1 ] ],
+      [
```
Comment (Xaleth) [inline data/json/recipes/recipe_food.json:4484]: To make bread flour it needs all-purpose flour and vital wheat gluten, but that isn't in the game yet, so this is my alternative for now.

File: data/json/recipes/recipe_food.json [hunk 3]
```diff
@@ -4468,6 +4468,30 @@
...
+    "result": "bread_flour",
+    "category": "CC_FOOD",
+    "subcategory": "CSC_FOOD_OTHER",
+    "skill_used": "cooking",
+    "difficulty": 2,
+    "time": "5 m",
+    "charges": 20,
+    "autolearn": true,
+    "batch_time_factors": [
+      [ [ "flour", 1 ] ],
```
Comment (Xaleth) [inline data/json/recipes/recipe_food.json:4483]: what does this error mean?

Comment (Xaleth) [reply data/json/recipes/recipe_food.json:4483]: from the github actions bot

**Comment by Maleclypse** [comment]
To fix your PR Summary under summary cut and paste this in  under the bolded SUMMARY
SUMMARY: Content "Add brown bread and bread flour"

**Comment by Xaleth** [comment]
done

**Comment by Maleclypse** [comment]
Files containers and breads need to be linted.  Cut and paste into https://dev.narc.ro/cataclysm/format.html  click lint then if no syntax errors pop up copy and paste the linted text back into the correct files.  Make sure you take the json file when linting.

**Pull request merged**